wirecard/oxid-ee
# WDPD-40: sofort payment

### This PR

* Adds Sofort. payment method to backend and shop
* Adds validation on payment method's config save action

### TODO

* Disable refund possibility 

### How to Test

* Activate Sofort. payment method and buy something with it
* Go to 'Shop Settings' -> 'Payment Methods' try to change the configuration of different payment methods, check if feedback is correct(e.g. there should be an error after you change the httppass to another invalid value and click save)
* test country code, should show error if not in a format like en_gb (de_de)
 
### Jira Links

https://jira.parkside.at/browse/WDPD-40
